tasks/ops-agent.yml

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
With three files like this, these tasks are getting mighty redundant. Is it possible to refactor this so that there aren't three separate scripts and yaml files? (If there's a good reason they're separate, feel free to explain.)

Reply:
Thats definitely possible, but it might be slightly less readable. We will need conditionals/variable substitution in lines 7, 23, 24, 32, 38, 45, 48 and 50.
Perhaps I can try that approach in a follow up PR after this gets merged?

Reply:
Ling and I sync'd offline that this would be best handled in a follow-up PR. So i'm going to merge this one for the time-being. Will send out the refactored PR by EOD.